df.viewscreen.xml

```diff
@@ -512,18 +512,19 @@
                 <int32_t/>
                 <int32_t/>
                 <stl-vector name='squads' type-name='int32_t'/>
+                <stl-vector name='messengers' type-name='int32_t'/> some strange huge values here but not pointers
```

Comment:
Sure it's not int16 or int8 or something, then?

Reply:
I think it's a bit wrong, I'll fix it now.

Reply:
So apparently `messengers` viewscreen field contains not histfigs but something else and I can't figure out what. The first field should be an id that gets written into this vector, and I can't find a class that would also result in the correct name displayed by the game.